MusicApp.SongService.Domain/Entities/Artist.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Text.Json.Serialization;
+
+namespace MusicApp.SongService.Domain.Entities;
+
+public class Artist
+{
+    public Guid Id { get; set; }
+    public string Username { get; set; } = string.Empty;
+    [JsonIgnore]
```

Comment:
?

Reply:
I dont understand the question.

Reply:
why did you add JsonIgnore attribute here?

Reply:
So that there are no loops during the output of the entity (the song contains an artist and the artist contains a list of songs).

Reply:
usage of dto's can prevent serialization errors